keyboards/crkbd/keymaps/rpbaptist/rgb.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "quantum.h"
+
+typedef union {
+    uint32_t raw;
+    struct {
+        bool     rgb_layer_indicator : 1;
+        bool     rgb_matrix_idle_anim : 1;
+        uint8_t  rgb_matrix_active_mode : 4;
+        uint8_t  rgb_matrix_idle_mode : 4;
+        uint8_t  rgb_matrix_active_speed : 8;
+        uint8_t  rgb_matrix_idle_speed : 8;
+        uint32_t rgb_matrix_idle_timeout : 32;
+    };
+} user_config_t;
+
+user_config_t user_config;
```

Comment:
This should be an extern, and declared in a c file.
```suggestion
extern user_config_t user_config;
```

Also, you may want: 
```c
_Static_assert(sizeof(user_config_t) == sizeof(uint32_t), "Userspace EECONFIG out of spec.");
```